docs/INSTALLATION.md

```diff
@@ -0,0 +1,72 @@
+# Installation #
+
+Use these instructions to install your Atlassian product using the Helm charts. Make sure you have followed the [Prerequisites guide](PREREQUISITES.md) before you proceed with the installation.
+
+#### 1. Add the Helm chart repository to your local Helm installation: ####
+
+Add the Helm chart repository to your local Helm installation:
+
+   `helm repo add atlassian-data-center https://atlassian-labs.github.io/data-center-helm-chart`
+
+   * Substitute `atlassian-data-center` with whatever name you wish
```

Comment:
I think we can drop this if people want to change it - they will probably know how to do it.

Reply:
Agree